src/components/Table/index.tsx

```diff
@@ -103,103 +103,15 @@ const HeaderButton = styled(Button)`
 	display: block;
 `;
 
-/**
- * Get the value specified by the `field` value
- * If a `render` function is available, use it to get the display value.
- */
-const renderField = <T extends {}>(row: T, column: TableColumn<T>): any => {
-	const value = get(row, column.field);
-
-	if (column.render) {
-		return column.render(value, row);
-	}
-
-	return value == null ? '' : value;
-};
-
-interface TableRowProps<T> {
-	className?: string;
-	isChecked: boolean;
-	isHighlighted: boolean;
-	keyAttribute: string | number;
-	onRowClick: (e: any) => void;
-	toggleChecked: (e: any) => void;
-	showCheck: boolean;
-	columns: Array<TableColumn<T>>;
-	href?: string;
-	data: T;
-	attributes?: React.AnchorHTMLAttributes<HTMLAnchorElement>;
-	checkboxAttributes?: React.InputHTMLAttributes<HTMLInputElement>;
-}
-
-class TableRow<T> extends React.Component<TableRowProps<T>, {}> {
-	shouldComponentUpdate(nextProps: TableRowProps<T>) {
-		const update = !isEqual(nextProps, this.props);
-
-		return update;
-	}
-
-	render() {
-		const {
-			attributes,
-			checkboxAttributes,
-			columns,
-			className,
-			data,
-			href,
-			keyAttribute,
-			isChecked,
-			isHighlighted,
-			showCheck,
-		} = this.props;
-
-		return (
-			<div
-				data-display="table-row"
-				data-highlight={isChecked || isHighlighted}
-				className={className}
-			>
-				{showCheck && (
-					<span data-display="table-cell">
-						<input
-							checked={isChecked}
-							data-key={keyAttribute}
-							onChange={this.props.toggleChecked}
-							{...checkboxAttributes}
-							type="checkbox"
-						/>
-					</span>
-				)}
-				{map(columns, column => {
-					const cellAttributes = isFunction(column.cellAttributes)
-						? column.cellAttributes(data, get(data, column.field))
-						: column.cellAttributes || {};
-					return (
-						<a
-							{...attributes}
-							href={href}
-							data-display="table-cell"
-							data-key={keyAttribute}
-							onClick={this.props.onRowClick}
-							{...cellAttributes}
-							key={column.field}
-						>
-							{renderField(data, column)}
-						</a>
-					);
-				})}
-			</div>
-		);
-	}
-}
-
 export default class Table<T> extends React.Component<
 	TableProps<T>,
 	{
 		allChecked: boolean;
-		reverse: boolean;
+		sort: {
+			reverse: boolean;
+			columnName: null | keyof T;
```

Comment:
`field` might be a better key than `columnName`, I have no strong opinions on this though.

Reply:
I feel like `field` is quite generic, but I do agree that it is better to be consistent with the rest of the usage.